src/page-analysis/content_script/extract-pdf-data.js

```diff
@@ -0,0 +1,58 @@
+// Returns a promise for an Object containing PDF Text and MetaData
+function getData(blob) {
```

Comment:
More meaningful name? We could actually expose just one `extractPdfData`, which accepts either a blob or a url, especially now url-to-blob adds just two lines of code.
```
export default async function extractPdfData({blob, url}) {
    if (blob === undefined) {
        response = await fetch(url)
        blob = await ...
    }
    ....
}
```